app/src/main/java/com/example/androidthings/assistant/EmbeddedAssistant.java

```diff
@@ -96,6 +97,9 @@
             new StreamObserver<AssistResponse>() {
                 @Override
                 public void onNext(final AssistResponse value) {
+                    if (DEBUG) {
+                        Log.d(DEBUG_TAG, value.toString());
```

Comment:
nit: not sure if you need a different TAG here, but some more context about the value you're printing would be nice.